ui/components/Navbar.tsx

```diff
@@ -50,8 +50,6 @@ const Navbar = ({ messages }: { messages: Message[] }) => {
       </div>
       <p className="hidden lg:flex">{title}</p>
 
-      <ThemeSwitcher size={17} className="lg:hidden ml-auto mr-4" />
-
```

Comment:
Should we also remove the `import ThemeSwitcher from './theme/Switcher';` at line 5? Same for `ui/components/EmptyChat.tsx:2` :)

Reply:
Although it doesn't matter much and will be auto removed by the compiler when on compiling stage, it's prefer to remove for code clean. I'll work on it no worries. Thanks for pointing it out